code/game/objects/items/toys.dm

```diff
@@ -1091,4 +1091,29 @@ obj/item/toy/cards/deck/syndicate
 	desc = "An adorable stuffed toy that resembles a space carp."
 	icon = 'icons/obj/toy.dmi'
 	icon_state = "carpplushie"
-	w_class = 2.0
\ No newline at end of file
+	w_class = 2.0
+
+/*
+ * Toy big red button
+ */
+/obj/item/toy/redbutton
+	name = "\improper Big Red Button toy"
+	desc = "A big, plastic red button. Who knows what it does?"
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "toybutton"

```

Comment:
Considering it uses the same sprite as the actual syndicate detonator it would be better to just use that by setting the icon to 'icons/obj/assemblies.dmi' and the icon_state to "bigred" as was done with the singulo toy's sprite for example.